src/test/scala/cromwell/ArrayAndMapIndexingWorkflowSpec.scala

```diff
@@ -0,0 +1,39 @@
+package cromwell
+
+import akka.testkit._
+import cromwell.binding.types.{WdlStringType, WdlArrayType}
+import cromwell.binding.values.{WdlInteger, WdlArray, WdlString}
+import cromwell.CromwellSpec.DockerTest
+import cromwell.util.SampleWdl
+
+import scala.language.postfixOps
+
+class ArrayAndMapIndexingWorkflowSpec extends CromwellTestkitSpec("ArrayAndMapIndexingWorkflowSpec") {
+  "A workflow with array/map indexes in expressions" should {
+    "run locally" in {
+      runWdlAndAssertOutputs(
+        sampleWdl = SampleWdl.ArrayAndMapIndexingWdl,
+        eventFilter = EventFilter.info(pattern = s"starting calls: test.echo_int, test.echo_str", occurrences = 1),
+        expectedOutputs = Map(
+          "test.echo_str.o" -> WdlString("bar"),
+          "test.echo_int.o" -> WdlInteger(200)
+        )
+      )
+    }
+    "run on docker" taggedAs DockerTest in {
```

Comment:
Is there a difference in expression handling between docker vs non-docker?

Otherwise, this test doesn't seem to add anything except checking that cromwell can still run workflows in docker.